### PR TITLE
Create table component

### PR DESCRIPTION
**Pull request by patjanawat**
Create default table component (No callapse, no expand, no pagging)

### CODE REVIEW

File: components/Table/Table.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
```
Comment (antho1404) [inline components/Table/Table.vue]: no need for a div here, you can just put the table as the root component

File: storybook/stories/Table.stories.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import { storiesOf } from '@storybook/vue'
+import Table from '@mesg-components/table/Table'
```
Comment (antho1404) [inline storybook/stories/Table.stories.js:2]: This library is not included in the package.json

File: components/Table/Table.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    <table>
+      <thead>
+        <tr>
+          <td
```
Comment (antho1404) [inline components/Table/Table.vue]:
```suggestion
          <th
```

File: components/Table/Table.vue [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            v-for="(header, i) in headers"
+            :key="i"
+            :style="textAlign(header.align)"
+          >{{header.text}}</td>
```
Comment (antho1404) [inline components/Table/Table.vue]:
```suggestion
          >{{header.text}}</th>
```

File: components/Table/Table.vue (hunk 1 above)
Comment (patjanawat) [reply components/Table/Table.vue]: Done.

File: storybook/stories/Table.stories.js (hunk 2 above)
Comment (patjanawat) [reply storybook/stories/Table.stories.js:2]: Done.

File: components/Table/Table.vue (hunk 3 above)
Comment (patjanawat) [reply components/Table/Table.vue]: Done.

File: components/Table/Table.vue (hunk 4 above)
Comment (patjanawat) [reply components/Table/Table.vue]: Done

**Review by antho1404 — APPROVED** [review]
there are still tiny issues with the design that doesn't exactly match the intended design but that let's merge that and we can improve later.

Good work, this version is much simpler

**Pull request merged**